metals/src/main/scala/scala/meta/metals/compiler/CompilerConfig.scala

```diff
@@ -4,6 +4,7 @@ import java.io.File
 import java.nio.file.Files
 import java.nio.file.Paths
 import java.util.Properties
+import scala.tools.nsc.settings.ScalaVersion
```

Comment:
Didn't know this existed!

Reply:
Me neither, I looked for it, since re-implementing that case class felt redundant.

Reply:
I probably would have cheated with using `scalaVersion.startsWith("2.12")` 😜

Reply:
Hehe, I don't trust strings 😅